ch_pipeline/processing/beam.py

```diff
@@ -121,7 +121,7 @@
 
 
 class HolographyFringestop(base.ProcessingType):
-    """"""
+    """ """
```

Comment:
why is this doctoring empty?

Reply:
these are not part of this PR. It's just bringing everything in line with the latest version of `black`.